tests/common/helpers/redis.py

```diff
@@ -0,0 +1,390 @@
+import logging
+
+
+class RedisCli(object):
+    """Base class for interface to RedisDb using redis-cli command.
+
+    Attributes:
+        host: a SonicHost or SonicAsic.  Commands will be run on this shell.
+        database: Redis database number.
+        pid: Port number of redis db.
+    """
+
+    def __init__(self, host, database=1, pid=6379):
+        """Initializes base class with defaults"""
+        self.host = host
+        self.database = database
+        self.pid = pid
+
+    def _cli_prefix(self):
+        """Builds opening of redis CLI command for other methods."""
+        # return "docker exec -i {docker} redis-cli -p {pid} --raw -n {db} ".format(
+        #     docker=self.docker, db=self.database, pid=self.pid)
+        return " -p {pid} --raw -n {db} ".format(db=self.database, pid=self.pid)
+
+    def _run_and_check(self, cmd):
+        """
+        Executes a redis CLI command and checks the output for empty string.
+
+        Args:
+            cmd: Full CLI command to run.
+
+        Returns:
+            Ansible CLI output dictionary with stdout and stdout_lines keys on success.
+            Empty dictionary on error.
+
+        """
+        result = self.host.run_redis_cli_cmd(cmd)
+
+        if len(result["stdout_lines"]) == 0:
+            logging.error("No command response: %s" % cmd)
```

Comment:
It would be better to initialize a logger than directly using the logging module.

Reply:
Fixed in 39f765f